opennav_docking/src/controller.cpp

```diff
@@ -190,6 +190,19 @@ bool Controller::isTrajectoryCollisionFree(
   return true;
 }
 
+geometry_msgs::msg::Twist Controller::computeRotateToHeadingCommand(double angle_to_target)
+{
+  // TODO(ajtudela): Replace with param
```

Comment:
TODO :-)

Reply:
Also, maybe this is better to do based on the max angular velocity parameter https://github.com/ros-navigation/navigation2/blob/main/nav2_docking/opennav_docking/src/controller.cpp#L49 & the accel / decel for rotating. We recently did this across Nav2 to better make sure we have dynamic feasibility in pure rotations or driving on heading